src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -1576,6 +1555,17 @@ void createConsumerNameIsNotSetAndBridgeIdIsSet(VertxTestContext context) throws
         consumerService()
             .deleteConsumer(context, groupId, consumerInstanceId[0]);
         context.completeNow();
+    }
+
+    @BeforeEach
+    void setUp() {
+        name = generateRandomConsumerName();
+        consumerWithEarliestReset.put("name", name);
```

Comment:
`consumerWithEarliestResetJson` to highlight that it's JSON as you did for `consumerJson`